test/functionals/api/v2/containers_controller_test.rb

```diff
@@ -109,6 +109,21 @@ class ContainersControllerTest < ActionController::TestCase
                                           :tag => tag }
             assert_response :created
           end
+
+          test 'creates a container with correct params' do
```

Comment:
Can you add another test for unwrapped parameters (by that I mean `post :create, { :compute_resource_id => ....}` without the top-level `:container =>` key)? 

It won't work right away, you'd have to add the equivalent of this `https://github.com/theforeman/foreman/pull/3864/files#diff-b167f4d1b1a23df4b43576b2dfacc1c2R7`  to the controller